build/package/windows/nri-amd64-installer/Product.wxs

```diff
@@ -49,47 +49,50 @@
 
     <Fragment>
         <DirectoryRef Id="PluginConfigsFolder">
-            <Component Id="CMP_V1_PLUGIN_CONFIGS" Guid="803C978C-B2A1-47C7-BB17-A2DE9E6D3145" KeyPath="yes">
+            <Component Id="CMP_V1_PLUGIN_CONFIGS" Guid="07bbd5f7-913c-46fa-b6c6-65ab2ca5f695" KeyPath="yes">

```

Comment:
Should we change all these componend GUID?
Can it cause issues? @davidbrota @brushknight